src/main/java/uk/gov/ons/fwmt/resource_service/exception/FWMTException.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.ons.fwmt.resource_service.exception;
+
+public class FWMTException extends Exception {
+
+  public enum Error {
```

Comment:
This should be replaced by ExceptionCode Enum but with correct error code format.